pkg/broker/api.go

```diff
@@ -436,37 +436,34 @@ func (b *AwsBroker) Update(request *osb.UpdateInstanceRequest, c *broker.Request
 	// Get the service instance
 	instance, err := b.db.DataStorePort.GetServiceInstance(request.InstanceID)
 	if err != nil {
-		desc := fmt.Sprintf("Failed to get the service instance %s: %v", request.InstanceID, err)
+		desc := fmt.Sprintf("Failed to get the service instance %q: %v", request.InstanceID, err)
```

Comment:
@jaymccon I noticed you used `%q` instead of `%s` somewhere else, and I like it in situations like this.